invenio_rdm_records/mappings/v6/records/record-v1.0.0.json

```diff
@@ -22,20 +22,60 @@
           "type": "keyword"
         },
         "$schema": {
-          "type": "keyword"
+          "type": "text",
+          "index": false
         },
         "recid": {
-          "type": "long"
+          "type": "keyword"
         },
         "title": {
-          "type": "text"
+          "type": "text",
+          "copy_to": "suggest_title"
+        },
+        "suggest_title": {
+          "type": "completion"
         },
         "description": {
           "type": "text"
         },
         "owners": {
           "type": "integer"
         },
+        "keywords": {
+          "type": "keyword"
+        },
+        "publication_date": {
+          "type": "date",
+          "format": "date"
+        },
+        "contributors": {
+          "type": "object",
+          "properties": {
+            "ids": {
+              "type": "object",
+              "properties": {
+                "source": {
+                  "type": "text"
```

Comment:
`keyword` might be nice here for aggregation purposes and since source will be from a limited set.

Reply:
:+1: